com.github.lxgr_linux.pokete.yml

```
@@ -0,0 +1,72 @@
+app-id: com.github.lxgr_linux.pokete
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: /app/bin/pokete
+
+finish-args:
+  # Store save files and logs
+  - "--filesystem=xdg-cache:create"
+  # Trading and update checking
+  - "--share=network"
+
+modules:
+  # Dependency of scrap engine; generated with flatpak-pip-generator
+  - name: python3-setuptools_scm
+    buildsystem: simple
+    build-commands:
+      - pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} "setuptools_scm>=6.4.2" --no-build-isolation
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/97/75/10a9ebee3fd790d20926a90a2547f0bf78f371b2f13aa822c759680ca7b9/tomli-2.0.1-py3-none-any.whl
+        sha256: 939de3e7a6161af0c887ef91b7d41a53e7c5a1ca976325f429cb46ea9bc30ecc
+      - type: file
+        url: https://files.pythonhosted.org/packages/6c/10/a7d0fa5baea8fe7b50f448ab742f26f52b80bfca85ac2be9d35cdd9a3246/pyparsing-3.0.9-py3-none-any.whl
+        sha256: 5026bae9a10eeaefb61dab2f09052b9f4307d44aee4eda64b309723d8d206bbc
+      - type: file
+        url: https://files.pythonhosted.org/packages/05/8e/8de486cbd03baba4deef4142bd643a3e7bbe954a784dc1bb17142572d127/packaging-21.3-py3-none-any.whl
+        sha256: ef103e05f519cdc783ae24ea4e2e0f508a9c99b2d4969652eed6a2e1ea5bd522
+      - type: file
+        url: https://files.pythonhosted.org/packages/e3/e5/c28b544051340e63e0d507eb893c9513d3a300e5e9183e2990518acbfe36/setuptools_scm-6.4.2-py3-none-any.whl
+        sha256: acea13255093849de7ccb11af9e1fb8bde7067783450cee9ef7a93139bddf6d4
+
+  # Dependency of pokete; generated with flatpak-pip-generator
+  - name: python3-scrap_engine
+    buildsystem: simple
+    build-commands:
+      - "pip3 install --verbose --exists-action=i --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} \"scrap_engine>=1.2.0\" --no-build-isolation"
+    sources:
+      - type: "file"
+        url: "https://files.pythonhosted.org/packages/3e/d6/b51ae9c311603c76e952a39baabca3c81887e4609f7bf7308c2e19ac8c9f/scrap_engine-1.2.0.tar.gz"
+        sha256: "767ffdc312b550777771cdc9a350a9f6dca73855259dd1fe7197c8736e38cac1"
+
+  # The actual game files
+  - name: pokete
+    buildsystem: simple
+    build-commands:
+      # Copy Files
+      - "cp -r . /app"
+      # Create the desktop entry
+      - "sed -i 's#Icon=/usr/share/pokete/assets/pokete.png#Icon=com.github.lxgr_linux.pokete#g' assets/pokete.desktop"
+      - "sed -i 's#Exec=/usr/bin/pokete.py#Exec=/app/bin/pokete#g' assets/pokete.desktop"
```

1. these are very wrong upstream. Notably `Icon` is *always* a name, never a path
2. the `Exec` should be just the command name, not an absolute path
3. `desktop-file-edit` is preferred over obscure sed commands, as it is more robust.

```suggestion
      - desktop-file-edit --set-icon=$FLATPAK_ID --set-key=Exec --set-value=pokete assets/pokete.desktop
```
(this should work regardless 1 and 2 being fixed later)